src/config/experimental.ts

```diff
@@ -8,13 +8,15 @@ import { STORAGE_IDS } from '@rainbow-me/model/mmkv';
  */
 
 export const LANGUAGE_SETTINGS = 'languageSettings';
+export const NOTIFICATIONS = 'Notifications';
 export const REVIEW_ANDROID = 'reviewAndroid';
 export const PROFILES = 'ENS Profiles';
 export const L2_TXS = 'L2 Transactions';
 
 export const defaultConfig = {
   [L2_TXS]: { needsRestart: true, settings: true, value: false }, // this flag is not reactive. We use this in a static context
   [LANGUAGE_SETTINGS]: { settings: false, value: false },
+  [NOTIFICATIONS]: { needsRestart: true, settings: true, value: false },
```

Comment:
```suggestion
  [NOTIFICATIONS]: { settings: true, value: false },
```

Do this needs restart?

Reply:
I think it does as the new sheet won't be propagated in when we press the toggle.